index.html

```diff
@@ -64,98 +73,106 @@ <h3 class="content-2">I am a Software Developer</h3>
         <section class="second-section">
                <div class="page-title">
                 <h2 class="section-title">My Recent Works</h2>
+                <div class="border-right"></div>
                </div>
              
               <div class="feature-wrapper">
-                <div><img src="./images/feature.png" alt=""></div>
-                <h2 class="project-title">Multi-Post Stories</h2>
-                <p class="project-discription">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's 
+                <div class="content-image-mobile"><img src="./images/feature.png" alt=""></div>
+                <div class="content-image-desktop"><img src="./images/Img-Placeholder.png" alt=""></div>
```

Comment:
- [ ] Kindly add to the `<div class="content-image-desktop">` some width like `width:40vw`  and to the `<img src="./images/feature.png" alt="">` width `100%`. because it makes overflows to your page.

![image](https://user-images.githubusercontent.com/35535130/155630305-d33a5b07-98b6-4ce5-87dc-e018b31651f0.png)